### PR TITLE
PR for #359

### PR DESCRIPTION
**Pull request by cjdb**
PR for #359

### CODE REVIEW

**Comment by ericniebler** [comment]
Could you address [this comment](https://github.com/ericniebler/stl2/issues/359#issuecomment-284258877)?

**Comment by cjdb** [comment]
Oops! That email must have slipped through. Will fix now.

**Comment by cjdb** [comment]
Addressed. Can you please review the `\rSec2` additions?

File: deprecated.tex [hunk 1]
```diff
@@ -32,7 +32,6 @@
 The following algorithms are deemed unsafe and are deprecated in this document.
```
Comment (CaseyCarter) [inline deprecated.tex]: Nitpick: let's change this to "algorithm signatures." It isn't the algorithms - e.g. `mismatch` - that are unsafe, it's these particular overloads.

File: deprecated.tex [hunk 2]
```diff
@@ -103,3 +98,109 @@
     transform(Rng&& rng1, I first2, O result,
               F binary_op, Proj1 proj1 = Proj1{}, Proj2 proj2 = Proj2{});
 \end{codeblock}
+
+\rSec2[depr.algo.mismatch]{Mismatch}
```
Comment (CaseyCarter) [inline deprecated.tex]: I think we can get away without all of these subsections if we add a single *Remarks* element like "These signatures behave as described in Clause~\ref{algorithms} with the name `last2` defined as `first2 + (last1 - first1)`."

**Comment by cjdb** [comment]
Ping. Not sure if emails come through for pull request commits (I don't own any awesome repos ;))

**Comment by CaseyCarter** [comment]
> Ping. Not sure if emails come through for pull request commits (I don't own any awesome repos ;))

Yes, they do:

image

This all looks great to me, I'll get it applied today for the post-Kona prospective working draft.

**Comment by CaseyCarter** [comment]
Thanks again, Christopher!

**Pull request merged**